core/src/main/java/com/scalar/db/storage/jdbc/JdbcTableMetadataManager.java

```diff
@@ -105,26 +113,291 @@ public JdbcTableMetadataManager(
-        + enclose("indexed", rdbEngine)
+        + enclose(INDEXED)
         + " FROM "
-        + enclosedFullTableName(schemaPrefix.orElse("") + SCHEMA, TABLE, rdbEngine)
+        + encloseFullTableNames(getMetadataSchema(), TABLE)
         + " WHERE "
-        + enclose("full_table_name", rdbEngine)
+        + enclose(FULL_TABLE_NAME)
         + " = ? ORDER BY "
-        + enclose("ordinal_position", rdbEngine)
+        + enclose(ORDINAL_POSITION)
         + " ASC";
   }
 
+  private String getInsertStatement(
+      String schema,
+      String table,
+      String columnName,
+      DataType dataType,
+      @Nullable KeyType keyType,
+      @Nullable Ordering.Order ckOrder,
+      boolean indexed,
+      int ordinalPosition)
+      throws SQLException {
+
+    return String.format(
+        "INSERT INTO %s VALUES ('%s','%s','%s',%s,%s,%s,%d)",
+        encloseFullTableNames(getMetadataSchema(), TABLE),
+        schemaPrefix.orElse("") + schema + "." + table,
+        columnName,
+        dataType.toString(),
+        keyType != null ? "'" + keyType + "'" : "NULL",
+        ckOrder != null ? "'" + ckOrder + "'" : "NULL",
+        computeBooleanValue(indexed),
+        ordinalPosition);
+  }
+
+  private String computeBooleanValue(boolean value) {
+    switch (rdbEngine) {
+      case ORACLE:
+      case SQL_SERVER:
+        return value ? "1" : "0";
+      default:
+        return value ? "true" : "false";
+    }
+  }
+
+  private String getDeleteTableMetadataStatement(String schema, String table) {
+    String fullTableName = schemaPrefix.orElse("") + schema + "." + table;
+    return "DELETE FROM "
+        + encloseFullTableNames(getMetadataSchema(), TABLE)
+        + " WHERE "
+        + enclose(FULL_TABLE_NAME)
+        + " = '"
+        + fullTableName
+        + "'";
+  }
+
+  private void insertMetadataColumn(
+      String schema,
+      String table,
+      TableMetadata metadata,
+      Connection connection,
+      int ordinalPosition,
+      String column)
+      throws SQLException {
+    KeyType keyType = null;
+    if (metadata.getPartitionKeyNames().contains(column)) {
+      keyType = KeyType.PARTITION;
+    }
+    if (metadata.getClusteringKeyNames().contains(column)) {
+      keyType = KeyType.CLUSTERING;
+    }
+
+    try {
+      String insertStatement =
+          getInsertStatement(
+              schema,
+              table,
+              column,
+              metadata.getColumnDataType(column),
+              keyType,
+              metadata.getClusteringOrder(column),
+              metadata.getSecondaryIndexNames().contains(column),
+              ordinalPosition);
+      execute(connection, insertStatement);
+    } catch (SQLException e) {
+      connection.rollback();
+      throw e;
+    }
+  }
+
+  private void createTableIfNotExists(Connection connection) throws SQLException {
+    String createTableStatement =
+        String.format("CREATE TABLE %s(", encloseFullTableNames(getMetadataSchema(), TABLE))
```

Comment:
It's a very minor comment.
I feel just calling one `String.format` makes the query more readable.
Or we can use do simple String concatenation or Guava's Joiner to remove redundant `String.format`.
https://github.com/google/guava/wiki/StringsExplained#joiner

Reply:
All right. I updated it using simple string concatenation.